leetcodeDownloader.py

```diff
@@ -26,7 +26,12 @@ def downloadSubmission(session, name, file_type, URL):
         return
     r = session.get(URL)
     code = re.search(r'submissionCode\:\ \'([^\']+)\'', r.content).group(1)
-    code = code.encode('utf8')
+
+    # address to possible UnicodeDecodeError for Chinese Characters
+    reload(sys)
+    sys.setdefaultencoding('utf8')
```

Comment:
I think it's not a very good way to deal with decode error in Python 2.
The data can be first decoded to unicode and then encode to utf-8 like following code.
`code = code.decode('utf-8')`
`s = json.loads('{"code": "%s"}' % code)`
...
`f.write(s['code'].encode('utf-8'))`